### ISSUE TITLE
Options for SwaggerUI like OAuthClientId and OAuthAdditionalQueryStringParams

### ISSUE THREAD

**Issue by sadiqkhoja** [vitalybibikov/AzureExtensions.Swashbuckle#47]
Is there a way to configure swaggerUI options like :
```
c.OAuthClientId(apiSettings.ClientId);
c.OAuthAdditionalQueryStringParams(new Dictionary<string, string>
{
    { "resource", $"{apiSettings.ClientId}"}
});
```

**Comment by vitalybibikov** [comment]
Hi, @sadiqkhoja 

I've never tried to.

Can you drop me a link to the docs and give an example why do you need them?

Thanks

**Comment by martin1cerny** [comment]
I would also appreciate if this was possible. I managed to configure your great extension to use our ID server, but UI requires [Client ID](https://swagger.io/docs/open-source-tools/swagger-ui/usage/oauth2/) every time which is annoying. Also, it would be good to be able to set different [oauth2RedirectUrl](https://swagger.io/docs/open-source-tools/swagger-ui/usage/configuration/) because there is no default static /oauth2-redirect.html file. However, it is easy enough to serve one. It might be a valuable resource in the static resources of this library as well, to keep things aligned.

**Comment by swasun99** [comment]
We are currently exploring to add Swagger for azure functions and this package helped solve our purpose, however would like to have OAuth enabled for user to be able to auth using Azure AD and make requests. any update if this feature is being worked on?

**Comment by martin1cerny** [comment]
Related pull request: https://github.com/vitalybibikov/AzureExtensions.Swashbuckle/pull/61

**Comment by vitalybibikov** [comment]
Released in 3.3.2
